megamek/src/megamek/common/IGame.java

```diff
@@ -796,6 +796,15 @@ static public Phase getPhaseFromName(@Nullable String name) {
      */
     public abstract Iterator<Entity> getEnemyEntities(final Coords c,
             final Entity currentEntity);
+    
+    /**
+     * Returns an <code>Enumeration</code> of active enemy entities
+     *
+     * @param currentEntity the <code>Entity</code> whose enemies are needed.
+     * @return an <code>Enumeration</code> of <code>Entity</code>s at the
+     *         given coordinates who are enemies of the given unit.
+     */
+    public abstract Iterator<Entity> getAllEnemyEntities(final Entity currentEntity);
```

Comment:
FYI, you don't need to use the public keyword in an interface since all interface methods are automatically public (yeah, all the methods in this file already have this mistake).